x-pack/plugins/security_solution/public/common/components/visualization_actions/configs/common/external_alert.ts

```diff
@@ -0,0 +1,150 @@
+          },
+        ],
+        yRightExtent: {
+          mode: 'full',
+        },
+        yLeftExtent: {
+          mode: 'full',
+        },
+      },
+      query: {
+        query: '',
+        language: 'kuery',
+      },
+      filters: [
+        {
+          meta: {
+            index: 'a04472fc-94a3-4b8d-ae05-9d30ea8fbd6a',
+            alias: null,
+            negate: false,
+            disabled: false,
+            type: 'phrase',
+            key: 'event.kind',
+            params: {
+              query: 'alert',
+            },
+          },
+          query: {
+            match_phrase: {
+              'event.kind': 'alert',
+            },
+          },
+          $state: {
+            store: 'appState',
+          },
+        },
+      ],
+      datasourceStates: {
+        indexpattern: {
+          layers: {
+            'a3c54471-615f-4ff9-9fda-69b5b2ea3eef': {
+              columns: {
+                '37bdf546-3c11-4b08-8c5d-e37debc44f1d': {
+                  label: '@timestamp',
+                  dataType: 'date',
+                  operationType: 'date_histogram',
+                  sourceField: '@timestamp',
+                  isBucketed: true,
+                  scale: 'interval',
+                  params: {
+                    interval: 'auto',
+                  },
+                },
+                '0a923af2-c880-4aa3-aa93-a0b9c2801f6d': {
+                  label: 'Count of records',
+                  dataType: 'number',
+                  operationType: 'count',
+                  isBucketed: false,
+                  scale: 'ratio',
+                  sourceField: '___records___',
+                },
+                '42334c6e-98d9-47a2-b4cb-a445abb44c93': {
+                  label: `Top values of ${stackByField}`, // could be event.category
+                  dataType: 'string',
+                  operationType: 'terms',
+                  scale: 'ordinal',
+                  sourceField: `${stackByField}`, // could be event.category
+                  isBucketed: true,
+                  params: {
+                    size: 10,
+                    orderBy: {
+                      type: 'column',
+                      columnId: '0a923af2-c880-4aa3-aa93-a0b9c2801f6d',
+                    },
+                    orderDirection: 'desc',
+                    otherBucket: true,
+                    missingBucket: false,
+                    parentFormat: {
+                      id: 'terms',
+                    },
+                  },
+                },
+              },
+              columnOrder: [
+                '42334c6e-98d9-47a2-b4cb-a445abb44c93',
+                '37bdf546-3c11-4b08-8c5d-e37debc44f1d',
+                '0a923af2-c880-4aa3-aa93-a0b9c2801f6d',
+              ],
+              incompleteColumns: {},
+            },
+          },
+        },
+      },
+    },
+    references: [
+      {
+        type: 'index-pattern',
+        id: 'security-solution-default',
```

Comment:
```suggestion
        id: `security-solution-${spaceName}`,
```
this should be dynamic

Reply:
Eventually all `id`  in the `references` will be replaced according to our dataViewId from sourcerer
x-pack/plugins/security_solution/public/common/components/visualization_actions/index.tsx Line 74 `useLensAttributes`, but I think it makes sense to make it clear in the configs as well, good point, thank you